### PR TITLE
Replace line-column with vfile-location

### PR DESCRIPTION
**Pull request by TrySound**
vfile-location is dependency free and quite fresh package.
https://github.com/vfile/vfile-location

line-column has outdated dependencies and was not touched in 5 years.
https://github.com/io-monad/line-column

### CODE REVIEW

File: lib/input.d.ts [hunk 1]
```diff
@@ -126,5 +126,7 @@ export default class Input {
    *
    * @param offset Source offset.
    */
-  fromOffset (offset: number): { line: number; col: number } | null
+  fromOffset (
+    offset: number
+  ): { offset?: number; line?: number; column?: number }
```
Comment (ai) [inline lib/input.d.ts:131]: We can’t change public API. Let’s convert object.

File: lib/input.js [hunk 2]
```diff
@@ -50,8 +50,14 @@ class Input {
...
 
   fromOffset (offset) {
-    let finder = lineColumn(this.css)
-    this.fromOffset = i => finder.fromIndex(i)
+    let finder = vfileLocation(this.css)
+    this.fromOffset = i => {
+      let position = finder.toPoint(i)
+      return {
+        line: position.line,
+        col: position.column
```
Comment (TrySound) [inline lib/input.js:58]: Maybe add TODO here to make api consistent in v9?

Comment (ai) [reply lib/input.js:58]: I am using issue for that

**Comment by ai** [comment]
LGTM. Will release it a few hours later.

**Comment by ai** [comment]
Released in 8.1.9. Thanks.

**Pull request merged**